container/worker/run_openqa_worker.sh

```diff
@@ -1,9 +1,19 @@
 #!/bin/bash
+# shellcheck disable=SC2012
 set -e
+set -x
```

Comment:
Leftover from debugging?

Reply:
I thought was interesting to provide more logs. But removed :)